ppr-ui/src/enums/registrationTypes.ts

```diff
@@ -167,10 +167,16 @@ export enum UIRegistrationClassTypes {
 
 export enum APIMhrDescriptionTypes {
   REGISTER_NEW_UNIT = 'REGISTER NEW UNIT',
-  SALE_OR_GIFT = 'SALE OR GIFT',
+  SALE_OR_GIFT = 'SALE / GIFT TRANSFER',
   CONVERTED = '** CONVERTED **'
 }
 
+export enum UIMhrDescriptionTypes {
+  REGISTER_NEW_UNIT = 'REGISTER NEW UNIT',
+  SALE_OR_GIFT = 'SALE OR GIFT',
```

Comment:
Shouldn't it match the the above enum from APIMhrDescriptionTypes ?

Reply:
That's what I thought at first too, but based on review, the UI display value is different than the API value for both CONVERTED AND SALE_OR_GIFT.

Reply:
Ok sounds good.